collection-scripts/gather_sriov

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+sriov_resources+=(ns/openshift-sriov-network-operator sriovnetworks sriovnetworknodestates sriovnetworknodepolicies sriovnetworks sriovoperatorconfigs network-attachment-definitions)
```

Comment:
There are two sriovnetworks.

Reply:
It would be needed to collect some additional sriov resources: sriovibnetworks, sriovnetworkpoolconfigs 
Other related resources are: nodes machineconfigs machineconfigpools featuregates kubeletconfigs ns/openshift-multus ns/openshift-marketplace (I assume these are already collected as part of must-gather).

There is also a must-gather script under sriov-network-operator repo: https://github.com/openshift/sriov-network-operator/blob/master/must-gather/collection-scripts/gather